ui/app/AppLayouts/Wallet/services/dapps/WalletConnectService.qml

```diff
@@ -137,13 +138,19 @@ QObject {
                     if (!dappsProvider.selectedAddress ||
                         (accountsInSession.includes(dappsProvider.selectedAddress)))
                     {
+                        dappFoundInWcSessions = true
                         wcSDK.disconnectSession(topic)
                     }
                 }
             }
-        });
 
-        root.revokeSession(url)
```

Comment:
@kounkou WDYT of this change? The intention here is not to delete the WC session from DB before trying to process it with the WC flows.
So the process is changed as follows:
- Try to find the session in the WC sessions.If the session is not found in the WC sessions then process it through the Browser connect impl

Reply:
I have no idea how to test the Browser connect though..Any help is much appreciated.